specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2020-10-01/bms.json

```diff
@@ -0,0 +1,313 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-10-01",
+    "title": "RecoveryServicesBackupClient",
+    "x-ms-code-generation-settings": {
+      "internalConstructors": false
+    }
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.RecoveryServices/vaults/{vaultName}/backupEncryptionConfigs/backupResourceEncryptionConfig": {
```

Comment:
>backupResourceEncryptionConfig [](start = 152, length = 30)

Is this the only name that is allowed? If yes, the typical singleton resource pattern is to use one of these pre-approved names: `default`, `main`, `current`, `latest`. In addition you should also have the ability to list all `backupEncryptionConfigs` (yes, even if there's only one).

Reply:
Looking into this.